django/src/rdwatch_scoring/models/satellite_fetching.py

```diff
@@ -5,3 +5,4 @@
 
 class SatelliteFetching(BaseSatelliteFetching):
     site = models.CharField(max_length=255)
+    model_run_uuid = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Is there ever a case where this would be null? And if not - we haven't deployed any of this code yet, so I wonder if it would be better to instead tweak the existing migration to include this field and set it to non-null. What do you think?

Reply:
Good call.  I did it because I didn't feel like clearing data out of my existing DB when testing. I can add this stuff to the previous migration and revert it back to remove the blank/null.